src/PrestaShopBundle/Translation/Provider/ModulesProvider.php

```diff
@@ -0,0 +1,91 @@
+<?php
```

Comment:
would you mind to add a test for this provider ? all providers are already tested, you can look at others tests it's really easy to to and time saver for the future :smile_cat: